core/src/main/java/org/apache/accumulo/core/conf/ClientProperty.java

```diff
@@ -53,10 +53,10 @@
   // BatchWriter
   BATCH_WRITER_MAX_MEMORY_BYTES("batch.writer.max.memory.bytes", "52428800",
       "Max memory (in bytes) to batch before writing"),
-  BATCH_WRITER_MAX_LATENCY_SEC("batch.writer.max.latency.sec", "120",
```

Comment:
Could use time units like `2s` or `150ms`.  Can use the function `ConfigurationTypeHelper.getTimeInMillis(String)` to do this.  This is what is used by server config.

Reply:
Do you think I should add types to keep track, like we have in Property?

Reply:
That would be nice if it avoids making each user of a config prop have to know how to parse it.